lib/cli/options.ts

```diff
@@ -31,22 +31,33 @@ export class Option {
 
   getNumber(): number {
     let value = this.getValue_();
-    if (value && typeof value === 'number') {
-      return +value;
+    if (value) {
```

Comment:
Good catch. My fix is for strings or numbers to return the `+value`.  For booleans, I plan to return `null`.